cmd/node/config/external.toml

```diff
@@ -13,3 +13,22 @@
     # EnabledIndexes represents a slice of indexes that will be enabled for indexing. Full list is:
     # ["tps", "rating", "transactions", "blocks", "validators", "miniblocks", "rounds", "accounts", "accountshistory"]
     EnabledIndexes    = ["tps", "rating", "transactions", "blocks", "validators", "miniblocks", "rounds", "accounts", "accountshistory"]
+
+# EventNotifierConnector defines settings needed to configure and launch the event notifier component
+[EventNotifierConnector]
+    # Enabled signals whether the driver is enabled or not
```

Comment:
this is a config. I think "signals" can easily confuse people. Might rename to 
 `# Enabled will turn on or off the event notifier connector`
valid for all comments.
Also, this is enabled. What happens if the connector can not connect to the ProxyUrl? Does it errors? The validators should need to have this enabled or not?

Reply:
Changed the comment. Set the `Enabled` flag to `false`, it shouldn't be active by default, good catch 👍🏻 
Validators don't need this enabled, the events driver is meant to be used with an observer. In case it can't connect to the ProxyUrl, it simply logs the error but doesn't break the node, as it simply returns.

Reply:
Great. :+1: Good to know!